interfaces/apparmor/template.go

```diff
@@ -566,6 +566,19 @@ var overlayRootSnippet = `
   "###UPPERDIR###/{,**/}" r,
 `
 
+var ptraceTraceDenySnippet = `
```

Comment:
The comment is very nice and informational. Thank you! I love that you included the commit ID for the relevant mainline bug fix.